sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/CommunicationCloudEnvironmentModel.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.android.communication.common;
+
+class CommunicationCloudEnvironmentModel {
```

Comment:
Can we move these (manually) generated types into an implementation folder so that they don't show up in the public interface?

(not sure how Java handles this)

Reply:
This class is package private so it won't show up in API